packages/packages/documents/src/components/top-bar/settings-button.tsx

```diff
@@ -0,0 +1,30 @@
+import { ToggleButton, Tooltip } from '@elementor/ui';
+import { __ } from '@wordpress/i18n';
+import { Document } from '../../types';
+import { openRoute, useRouteStatus } from '@elementor/v1-adapters';
+import CogIcon from '../../icons/cog-icon';
+
+type Props = {
+	type: Document['type']
+}
+
+export default function SettingsButton( { type }: Props ) {
+	const { isActive, isBlocked } = useRouteStatus( 'panel/page-settings' );
+
+	/* translators: %s: Post type label. */
+	const title = __( '%s settings', 'elementor' )
```

Comment:
This is the product requirement? always putting the word "settings" at the end? a bit weird..

Reply:
Yeah it is